.gitlab/e2e/e2e.yml

```diff
@@ -29,6 +29,7 @@
     - GOOGLE_CREDENTIALS=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $E2E_TESTS_GCP_CREDENTIALS) || exit $?; export GOOGLE_CREDENTIALS
     # Generate external links to CI VISIBILITY, used by artifacts:reports:annotations
     - inv -e gitlab.generate-ci-visibility-links --output=$EXTERNAL_LINKS_PATH
+    - echo "This job is owned by $(inv owners.find-jobowners --job $CI_JOB_NAME)"
```

Comment:
I think this does not work well with, https://github.com/DataDog/datadog-agent/pull/30409 - this line will always show `@DataDog/multiple`.

Reply:
Indeed: https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/686101229#L97

This will make IH life more complicated no?

Reply:
Yes, but it is correct (the jobs' execution is owned by DevX Loops, the tests inside by the team that wrote the test). 

Personally, I'd be in favor of replicating what the KMT tests do: they show the owner of the test alongside the test name in the failure summary ([code](https://github.com/DataDog/datadog-agent/blob/bbfd6c5b2fe1ed9b17ac61b52e63bba1cfa6ed7a/test/new-e2e/system-probe/test-json-review/main.go#L216), [example](https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/682556303#L3168)).